File: C++/Easy/Accepted/crawler-log-folder_6.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        
        int n = logs.size();
        int ans = 0;
        for (int i = 0; i < n; i++) {
            if (logs[i] == "../") {
                ans = max(ans - 1, 0);
            } else if (logs[i] == "./") {
                continue;
            } else {
                ans++;
            }
        }
        return ans;
    }
};

File: Rust/Easy/WrongAnswer/crawler-log-folder_6.rs
impl Solution {
    pub fn min_operations(logs: Vec<String>) -> i32 {
        
        let mut ans = 0;
        for s in logs {
            if s == "../" {
                ans = ans.max(0) - 1;
            } else if s != "./" {
                ans += 1;
            }
        }
        ans
    }
}